src/core/document.js

```diff
@@ -820,6 +821,47 @@ class PDFDocument {
     });
   }
 
+  get xfaDatasets() {
```

Comment:
I cannot help but noticing the similarities between this new method and the existing `xfaData` one just below. Hence I wonder if it'd perhaps make sense, in a follow-up PR, to re-factor these two method to remove some of the duplication?

Reply:
Yes you're right, I'll do a follow-up PR.